CLAUDE.md

```
@@ -508,15 +503,25 @@ From `pulumicost.v1.ErrorCode`:
+- [Understanding NLCU Dimensions](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancer-capacity-units.html)
+- NLCU = max of (new connections/sec, active connections, processed bytes)
 
-- `resource_type`: "s3", "lambda", "rds", "dynamodb"
-- Supports() returns `supported=true` with `reason="Limited support - returns $0 estimate"`
-- GetProjectedCost() returns:
-  - `unit_price=0`
-  - `cost_per_month=0`
-  - `currency="USD"`
-  - `billing_detail="<Service> cost estimation not implemented - returning $0"`
+**Quick Reference:**
+| Workload Type | Typical ALB LCU | Typical NLB NLCU |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)